nautobot_floor_plan/tests/test_models.py

```diff
@@ -45,6 +46,30 @@ def test_create_floor_plan_invalid_duplicate_location(self):
         with self.assertRaises(ValidationError):
             models.FloorPlan(location=self.floors[0], x_size=2, y_size=2).validated_save()
 
+    def test_origin_seed_increase(self):
+        """Test that existing tile origins are updated during origin_seed updates"""
+        floor_plan = models.FloorPlan.objects.create(
+            location=self.floors[0], x_size=3, y_size=3, x_origin_seed=1, y_origin_seed=1
+        )
+        tile_1_1_1 = models.FloorPlanTile(floor_plan=floor_plan, x_origin=1, y_origin=1, status=self.status)
+        tile_1_1_1.validated_save()
```

Comment:
Let's make sure the test covers multiple tiles that may potentially collide mid-update as in my other comment.